src/api/authApi.ts

```diff
@@ -0,0 +1,142 @@
+import axiosInstance from '@/api/axiosInstance';
+
+export interface LoginRequestData {
+  email: string;
+  password: string;
+  rememberMe: boolean;
+}
+
+export interface CheckEmailCodeData {
+  email: string;
+  authCode: string;
+}
+
+export interface FindEmailData {
+  name: string;
+  phoneNumber: string;
+}
+
+export interface CheckEmailCodeForPasswordData {
+  email: string;
+  authCode: string;
+}
+
+export interface ResetPasswordData {
+  emailAuthCode: string;
+  email: string;
+  password: string;
+  rewritePassword: string;
+}
+
+// 로그인 API
+export const login = async (loginData: LoginRequestData) => {
+  const response = await axiosInstance.post('/v1/auth/login', loginData);
+  return response.data;
+};
+
+// Token 유효성 검증 및 회원 정보 조회 API
+export const checkAuth = async () => {
+  const response = await axiosInstance.get('/v1/auth');
+  return response.data;
```

Comment:
P4;
당연히 아시겠지만.. 요런식으로도 가능합니다 :)

```suggestion
  const { data } = await axiosInstance.get('/v1/auth');
  return data;
```